tests/congruence_tests/test_common.py

```diff
@@ -372,13 +372,21 @@ def init_client(
     collection_name: str = COLLECTION_NAME,
     vectors_config: Optional[Union[dict[str, models.VectorParams], models.VectorParams]] = None,
     sparse_vectors_config: Optional[dict[str, models.SparseVectorParams]] = None,
+    init_debug_collection: bool = False,
 ) -> None:
     initialize_fixture_collection(
         client=client,
         collection_name=collection_name,
         vectors_config=vectors_config,
         sparse_vectors_config=sparse_vectors_config,
     )
+    if init_debug_collection:
+        initialize_fixture_collection(
+            client=client,
+            collection_name=collection_name + "_debug",
+            vectors_config=vectors_config,
+            sparse_vectors_config=sparse_vectors_config,
+        )
```

Comment:
it seems like all the changes in this file are unnecessary

Reply:
I would, may be change the name of this collection, but it is the collection I write vectors if they break tests, so we could easily reproduce scenario